src/main/java/com/Kh033Java/travelbook/validation/ValidationUtil.java

```diff
@@ -8,9 +8,9 @@
 
     private static String template = "%s with such id [%s] not found";
 
-    public static <T> T checkBeforeGet(final Optional<T> obj, final Long id, final Class clazz) {
-        if (obj.isEmpty()) {
-            throw new NotFoundException(String.format(template, clazz.getSimpleName(), id));
+    public static <T> T checkBeforeGet(final Optional<T> obj, final String login, final Class clazz) {
+        if (obj.isPresent()) {
+            throw new NotFoundException(String.format(template, clazz.getSimpleName(), login));
```

Comment:
this method and class is never used

Reply:
We need finally clarify about this class, I hope that we will resolve this problem in nearest future.